src/probnum/random_variables/_normal.py

```diff
@@ -174,9 +176,9 @@ def __init__(
             var = self._dense_var
             entropy = self._dense_entropy
 
-            if cov_cholesky is None:
-                self._compute_cov_cholesky = self.dense_cov_cholesky
-            else:
+            self._compute_cov_cholesky = self.dense_cov_cholesky
```

Comment:
even if cov_cholesky is provided in the ``__init__``, this remains the default behaviour for dense or cov_operator settings (see line 165). Now, even if a Cholesky factor is provided, calling `Normal.precompute_cov_cholesky()` directly computes a new one

Reply:
Why is that?

Reply:
I guess this is resolved in the discussion below?

Reply:
Yup, thanks for clarifying!